docs/pages/eas-update/how-it-works.mdx

```diff
@@ -104,7 +104,7 @@ Now that we're familiar with the core concepts of EAS Update, let's talk about h
 
 When an Expo project that includes `expo-updates` is built the included native Android and iOS code is responsible for managing, fetching, parsing, and validating updates.
 
-When the library checks for updates and when it downloads them is [configurable](/versions/latest/config/app.mdx#updates). By default, the library will check for an update when it is opened. If an update newer than the current running update is found, it will download and run the newer update. If the library does not find a newer update, it will instead run the newest downloaded update, falling back to the update that was embedded inside the app at build time if none have been downloaded.
+When the library checks for updates and when it downloads them are [configurable](/versions/latest/config/app.mdx#updates). By default, the library will check for an update when it is opened. If an update newer than the current running update is found, it will download and run the newer update. If the library does not find a newer update, it will instead run the newest downloaded update, falling back to the update that was embedded inside the app at build time if none have been downloaded.
```

Comment:
```suggestion
When the library checks for updates and when it downloads them are [configurable](/versions/latest/config/app.mdx#updates). By default, the library will check for an update when it is opened. If an update newer than the currently running update is found, it will download and run the newer update. If the library does not find a newer update, it will instead run the newest downloaded update, falling back to the update that was embedded inside the app at build time if none have been downloaded.
```